website/orders/api/v1/views.py

```diff
@@ -52,7 +52,7 @@ class OrderListCreateAPIView(ListCreateAPIView):
 
     def get_queryset(self):
         """Get the queryset."""
-        return self.queryset.filter(shift=self.kwargs.get("shift"))
+        return self.queryset.filter(shift=self.kwargs.get("shift")).order_by("-prioritize", "created")
```

Comment:
Don't we want to set this order_by in the `Meta` of `Order`?

Reply:
I wouldn't, because it will display the ordering differently in the admin.

Reply:
Good point